build.cake

```diff
@@ -677,13 +677,13 @@ Task("TestConsole")
 Task("TestEngineCore")
     .Description("Builds and tests the engine core assembly")
     .IsDependentOn("TestNet20EngineCore")
-    .IsDependentOn("TestNetStandard20EngineCore")
+    //.IsDependentOn("TestNetStandard20EngineCore")
     .IsDependentOn("TestNetCore31EngineCore");
 
 Task("TestEngine")
     .Description("Builds and tests the engine assembly")
     .IsDependentOn("TestNetFxEngine")
-    .IsDependentOn("TestNetStandard20Engine")
+    //.IsDependentOn("TestNetStandard20Engine")
```

Comment:
Before checking in I think we should just remove this outright unless we expect that this is something we'll want to add back in the future.

Reply:
Leaving as a reminder to discuss this. I don't see the need for a .NET Standard engine but I wanted confirmation from others. I'll add a  TODO comment to make that clear.

Reply:
A library can be written against the standard.
Actual testing has to be done on a real platform.
If we need different versions of the engine for the different platforms, then a netstandard engine has no benefit as it won't be used.

Reply:
I'm pretty sure we won't want a .NET Standard engine in the future. Even if we abandoned NETFX for the console runner, I think we would want to use .NET Core, probably 8.0, for the engine.

In any case, dropping or keeping it is a policy question and needs a bit more discussion at some point.